### ISSUE TITLE
Consider to rename the "Subhead" block 

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5790]
As a user, I wouldn't have a clear understanding of what a "Subhead" block is, see #2091. I think this block is really useful for better typography and I really like it. I think in typography its name its name varies, depending on the country and actual usage, sometimes it's called "Summary" sometimes "Lead paragraph" (in my language it has a different name).

Worth considering also translations: not sure there's a suitable translation for "Subhead" in all languages, while "Summary" or "Lead paragraph" are easily translatable.

**Comment by SuperGeniusZeb** [comment]
I agree that "Subhead" is confusing. When I first saw the block I thought it might be for some kind of heading, but when I tried to use it I was confused when it seemed to just be like paragraph. I think something like "Summary" or "Lead Paragraph" would be more clear.

**Comment by karmatosed** [comment]
This is interesting as yes I agree that the title does maybe not say what it is, we totally can do better. I think 'Subheading' works: https://dictionary.cambridge.org/dictionary/english/subheading. This seems to say what it does.

**Comment by afercia** [comment]
> 'Subheading' works

Hm not so sure that would work well for translations. Typographic terms vary depending on the culture and language. I'm afraid I don't know what the proper English term is, I've found `Intro, Deck, Standfirst, Kicker, Nut Graph`, see also the ones mentioned here: http://www.magazinedesigning.com/magazine-page-elements/

In my language, "subheading" would be translated as "Sotto titolo" which means a smaller title under the main title. Instead, this is a paragraph with bigger font size, italic style, meant to be used as an "intro": that's what we call "Sommario" (Summary).

**Comment by karmatosed** [comment]
As a subheading literally means:

> a word, phrase, or sentence that is used to introduce part of a text

That's what this is. Summary or intro means something this isn't. I know it's a balance but lets go with that for now with it being 'Subheading'.

**Comment by afercia** [comment]
> As a subheading literally means:

Maybe true in English :) I've tried to explain that would be confusing for translations in other languages, which I think is an important point given WordPress is a global project. Fair enough though.

**Comment by karmatosed** [comment]
@afercia I totally agree having words that mean to others in any culture is important. My comment was also based on your stating of what it means in Italian:

> In my language, "subheading" would be translated as "Sotto titolo" which means a smaller title under the main title.

Which actually works in this case.

**Comment by karmatosed** [comment]
I think going for subtitle makes sense here at least as a balance between languages. I'll work on a PR for this, thanks.

**Comment by mrwweb** [comment]
I still have some concerns about "subheading" which I think get to a second part of @afercia's OP:

> it's called "Summary" sometimes "Lead paragraph"

Subheading could very reasonably be misconstrued for "Heading 2" (the heading "below" the main heading). "Heading", obviously but notably, appears in the word "subheading"!

I do a huge about of education around proper heading usage and this is almost sure to confuse people I've trained. Regardless of what the dictionary definition of "subheading" is, I fear the real-world implications of this choice will lead to misuse of the style.

Most importantly, though, I think the point of OP is that the usage of this style is actually broader than simply a "subheading" and could be used for a full intro paragraph (or "lede" in journalism lingo). Thinking about the common use-cases I run into when responding to user requests for formatting styles, there are two separate requests that this style seems to merge at least in labeling:

1. A subtitle (i.e. a _de-emphasized_ secondary segment of the page title)
2. An "Intro Paragraph" (e.g. an _emphasized_ paragraph at the start of an article)

I'd encourage a rethinking of this style's name to reflect that #2 usage, _especially_ if the style is to be implemented with a class on a paragraph. (The first time I used this style, I assumed it would be a class applied to a heading.)

**Comment by karmatosed** [comment]
I'd totally be up in it being subtitle if everyone agrees.

**Comment by mrwweb** [comment]
tl;dr - "Subtitle" works but doesn't seem to be an accurate description of the block's current usage.

--------------------------

Is "Subtitle" an accurate description of the block's intent? If so, that immediately raises a number of questions in my mind that don't seem in line with this block (or potentially any block):

1. Is a paragraph element the most appropriate output? Should it instead be appended to the Heading 1 title inside a `<span>`? Generally speaking a page's title should be the Heading 1 and a Heading 2 is not appropriate markup for a subtitle.
1. Should the user be limited to a single Subtitle block on a page? (i.e. If there's only one page title, what does it mean to have multiple subtitles?)
1. Should the user be limited to having the subtitle block only be the first block? (What does it mean if the subtitle is the last block on the page?)

Depending on some of those answer, a "Subtitle" field feels much more appropriate as a meta-field/setting than a block which can have multiple instances.

I think this kind of this is super important because the name has to communicate the intended use if themes are to be able to design block styles that will make content truly portable. If one theme treats a block as a subtitle but another treats it as "Introduction Text", the user will get confused.

(Also, I don't know (as in, I have no idea about) how translatable "Subtitle" is.)

**Comment by karmatosed** [comment]
A few of those reasons are why it was called subheading as then it can have more than one use.

> a heading given to a subsection of a piece of writing.
"the page is broken up into short paragraphs with sub-headings"

If you look at an example like Google docs, subtitle can be used more than once, as a result I think there is good ground to do that and have it called that. I also think it's fair to have a p and not be a h1 or any other styling.

**Comment by SuperGeniusZeb** [comment]
To remove any confusion over which HTML element is semantically correct, here is what the W3C HTML 5.2 specification and HTML 5.3 working draft says:

>h1–h6 elements must not be used to markup subheadings, subtitles, alternative titles and taglines unless intended to be the heading for a new section or subsection. Instead use the markup patterns in the [§4.13 Common idioms without dedicated elements](https://www.w3.org/TR/html53/common-idioms-without-dedicated-elements.html#common-idioms-without-dedicated-elements) section of the specification.

From https://www.w3.org/TR/html53/sections.html#the-h1-h2-h3-h4-h5-and-h6-elements

See examples at https://www.w3.org/TR/html53/common-idioms-without-dedicated-elements.html#common-idioms-without-dedicated-elements

It looks like a `<p>` element is almost always the correct choice for the use cases of the Subhead/Subheading/whatever-it-will-be-called block.

**Comment by mrwweb** [comment]
I guess my big concern is that users will use subheadings _instead_ of headings and/or use it for it's visual appearance than as a heading. Headings have an important semantic meaning that provides a page outline for assistive technology and the subhead is a paragraph. That means an unclear name or interface that causes confusion could actively impede people making their content acccessible.

When thinking both about translations and in just one language, it seems the most important thing for any block name is an alignment of:

1. Page editors' understanding of the intended usage
1. The semantic output in the source
1. The likely range of visual displays so that porting content from one theme to another results in an expected output for users

As designed and labels, the text style doesn't scream "subheading" to me, and I'm unclear if that's even an important enough style to justify core inclusion. Knowing users, I'd be shocked if many people don't treat is as a means of text formatting.

It seems like there's just not clarity around the intended purpose of this style, even in the original ticket #2091 that added it where it truly started as a Sub**title**. **If we can establish the intended use case for the block, then everything else (UI, design, label) should stem from there.** One was suggested when this block was added, but things have clearly strayed from there.

I still think an "Introductory Text" style is clearer (meeting all three criteria above) and more useful, but if that's not what people are going for, then I recognize my comments aren't quite relevant.

**Comment by aaronjorbin** [comment]
`Subtitle`
Sweet, that's what I add to my video blocks so they are understandable by people in other languages? 
---
If the goal is for this to be the first paragraph of text and for it to have a slightly different style than the rest of the content, then `Dek` is the most technically accurate term, but it's also going to be the one that is least likely to be understood.

I'm going to propose something else, let's get rid of this block. Is this a block that we expect 80% of sites to make use of?

**Comment by karmatosed** [comment]
Lots of feedback not this which is great.

Firstly, this block is already one that I've seen people really like, so for now keeping it makes sense. Let's put removing it over into the 'maybe' pile. I'm not ever ruling out any block not making version one.

That said, let's see if we can work through this feedback and get the PR that already was in progress moved on. The thing that right now is holding it up is the naming. Reading the feedback 2 options stand out:

- Subheading: the original target of the PR.
- Subtitle: the new suggestion.

@aaronjorbin whilst I understand the point you are making about subtitle (I felt it to initially), this is the name used in other examples like Google.  I also agree that 'Dek' right now isn't understood enough to use.

I see both subheading and subtitle has having pros and cons, all outlined previously. If I had to standby anything I'd say because of the use of subtitle in other applications that for me is the one to change to right now.

**Comment by gziolo** [comment]
Can we make the final decision and proceed? Both proposals are better than what we have at the moment ...

**Comment by karmatosed** [comment]
@gziolo yes, lets go with subtitle on this.

**Comment by karmatosed** [comment]
Because of keyword issues on headings where subtitle is a keyword already, for now lets go with subheading and close this. It is a fairly easy change to iterate on.

**Comment by gziolo** [comment]
Updated in #6407.
